core/src/avm1/object_reference.rs

```diff
@@ -131,7 +138,7 @@ impl<'gc> MovieClipReference<'gc> {
                 // We have to fallback to manual path-walking if the object is removed
                 if !mc.read().display_object.removed() {
                     let display_object = mc.read().display_object;
-                    let display_object = Self::process_swf5_references(activation, display_object);
+                    let Some(display_object) = Self::process_swf5_references(activation, display_object) else { return None; };
```

Comment:
Could this not be ```let display_object = Self::process_swf5_references(activation, display_object)?;``` (this same change is in a few places)

Reply:
Whoops, completely missed this. Fixed in latest commit.